services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/PushStatusCollector.java

```diff
@@ -190,10 +194,11 @@ private void scanDaVinciPushStatus() {
       }
 
       LOGGER.info(
-          "Received DaVinci status: {} with details: {} for topic: {}",
+          "Received DaVinci status: {} with details: {} for topic: {}. Server status: {}",
           daVinciStatus.getStatus(),
```

Comment:
why not just print `daVinciStatus` here?

Reply:
Done